### PR TITLE
Week1 Draft 1 of Problem Set and Answer Key

### PR DESCRIPTION
**Pull request by splazatorres**

### CODE REVIEW

**Comment by splazatorres** [comment]
Binder link for Draft 1 <a href='https://mybinder.org/v2/gh/CUB-Computational-Tools/2020-ps-splazatorres/binder-R?urlpath=git-pull?repo=https%3A%2F%2Fgithub.com%2FCUB-Computational-Tools%2F2020-ps-splazatorres%26branch%3Dweek1%26urlpath%3Drstudio'></a>

File: week1.Rmd [hunk 1]
```diff
@@ -14,5 +14,140 @@ output:
...
+
+## Background
+
+*Avogadro's Law was hypothesized by Amedeo Avogadro in 1812.*
+
+Avogadro's Law states that:
+
+> Equal volumes of any gas, at the same temperature and pressure, contain the same number of molecules.
+
+In this statement the **number of molecules** refers to the **number of moles**. Moles measure the amount of a substance.
```
Comment (KevRozmiarek) [inline week1.Rmd]: How much is in a mole?

File: week1.Rmd [hunk 2]
```diff
@@ -14,5 +14,140 @@ output:
...
+or 
+
+$$
+\frac{V_{1}}{n_{1}}\ = \frac{V_{2}}{n_{2}}\ 
+$$
+where $n$ is the number of moles of gas, $V$ is the volume of gas in liters, and $k$ is a constant that can be calculated.
+
+## Examples
+
+You can calculate the $k$ constant by deriving it from the ideal gas law equation where $P$ is pressure in pascals, $R$ is the gas constant, and $T$ is the Kelvin temperature:
```
Comment (vaga3461) [inline week1.Rmd:71]: I like this format of defining symbols (P, R, etc) before they're used in an equation. V and n (above) could be defined earlier for clarity.

File: week1.Rmd [hunk 3]
```diff
@@ -14,5 +14,140 @@ output:
...
+$$
+We can see that $V_{2}$ is larger than $V_{1}$ because it contains more moles of gas.
+
+## Questions
+
+Q1. What does $V_{2}$ mean in the context of this equation:
+$$
+\frac{V_{1}}{n_{1}}\ = \frac{V_{2}}{n_{2}}\ 
+$$
+Q2. If you have a 22 liters of oxygen gas that contain 3 mol of oxygen, how many mol of gas would 30 liters of oxygen have?
```
Comment (KevRozmiarek) [inline week1.Rmd]: Maybe clarify that this is at STP?

**Review by KevRozmiarek — CHANGES_REQUESTED** [review]
Honestly it looks great. No need to change much unless you want to elaborate on topics but we aren't actually teaching this soooo... maybe not.

File: week1.Rmd [hunk 4]
```diff
@@ -14,5 +14,140 @@ output:
...
+\begin {align}
+V & = 33.256 \times 1\ \\
+V & = 33.256\ liters
+\end{align}
+$$
+But if we have 2 mol of oxygen gas the volume would be:
+
+$$
+\begin {align}
+V & = 33.256 \times \ \\
```
Comment (vaga3461) [inline week1.Rmd]: missing a 2 here after the multiplication symbol

File: week1.Rmd [hunk 5]
```diff
@@ -14,5 +14,140 @@ output:
...
+V & = 66.512\ liters
+\end{align}
+$$
+
+Another example: 
+
+If 6 liters have 3 mol of hydrogen gas then how many liters 6 mol of hydrogen gas have?
```
Comment (vaga3461) [inline week1.Rmd]: This question could read more easily if it were a little longer/rearranged: "If 3 mol of hydrogen gas occupies a volume of 6 liters, then how many liters will 6 mol of hydrogen gas occupy?"

**Review by vaga3461 — CHANGES_REQUESTED** [review]
overall looks good, just a couple small things for clarification.!

File: week1.Rmd (hunk 5 above)
Comment (splazatorres) [reply week1.Rmd]: Thanks for the rewrite!

**Pull request merged**